--- obex-data-server-0.4.6/src/ods-obex.c	2011-02-10 10:14:42.000000000 +0100
+++ obex-data-server-0.4.6/src/ods-obex.c	2013-12-07 22:14:10.611430934 +0100
@@ -412,7 +412,7 @@
 		goto err;
 	}
 
-	interfaces_num = OBEX_FindInterfaces(obex_context->obex_handle, &obex_intf);
+	interfaces_num = OBEX_EnumerateInterfaces(obex_context->obex_handle);
 	if (intf_num >= interfaces_num) {
 		g_set_error (error, ODS_ERROR, ODS_ERROR_FAILED, "Invalid interface number");
 		goto err;
Binarne pliki ./obex-data-server-0.4.6/src/.ods-obex.c.un~ i ../obex-data-server-0.4.6/src/.ods-obex.c.un~ różnią się
